src/PKSim.Core/Services/ParameterUpdater.cs

```diff
@@ -153,5 +153,11 @@ private bool areValuesEqual(IParameter targetParameter, IParameter sourceParamet
       {
          return ValueComparer.AreValuesEqual(targetParameter, sourceParameter);
       }
+
+      private bool areDisplayUnitsEqual(IParameter targetParameter, IParameter sourceParameter)
+      {
+         //We compare name are some dose related units may be the representing the same unit but with different reference
+         return string.Equals(targetParameter.DisplayUnit?.Name, sourceParameter.DisplayUnit?.Name);
```

Comment:
@Yuri05 I updated this method here because I realzied that a bunch of commands were created for nothing. For some reason, we have different instances of the same unit (only for our home made units for dose). That way, we do not create stupid commands for nothing